lib/trento_web/controllers/v1/activity_log_controller.ex

```diff
@@ -60,7 +62,12 @@ defmodule TrentoWeb.V1.ActivityLogController do
     ]
 
   def get_activity_log(conn, params) do
-    with {:ok, activity_log_entries, meta} <- ActivityLog.list_activity_log(params) do
+    {current_function, _} = __ENV__.function
+    user = conn.assigns.current_user
```

Comment:
Current user should be retrieved using Pow primities and pow context.
This is useful because we can change the pow configuration or update the pow library and be able to retrieve the current user, the fact that is hardwired into the assigns with `current_user` as key is a pow config implementation detail.

Use

` %User{} = user = Pow.Plug.current_user(conn)`